tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -71,17 +71,30 @@ public function test_wp_enqueue_script() {
 	 */
 	public function test_non_standalone_and_standalone_after_script_combined() {
 		// If the main script with defer strategy has a `after` inline script; expected one with type='javascript' and other with type='module'.
```

Comment:
```suggestion
		// If a main script with a `defer` strategy has an `after` inline script, we expect one instance with type='javascript' and another with type='text/template'.
```

@kt-12 there is no `type="module"` within our work spec, only `type="text/template"`